app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -739,8 +739,10 @@ public function processPageHit(Hit $hit, $page, Request $request, Lead $lead, $t
      */
     public function getHitQuery(Request $request, $page = null)
     {
-        // get all post params
-        $query = $request->request->all();
+        $get  = $request->query->all();
```

Comment:
@kuzmany what is the difference between this and getQueryFromUrl() on line 752

Reply:
I suppose $query['page_url'] to getQueryFromUrl method is already without ct param in url